sdk/servicebus/service-bus/test/streamingReceiver.spec.ts

```diff
@@ -1214,7 +1214,7 @@ describe("Streaming - onDetached", function(): void {
     await receiverIsActive;
 
     // Simulate onDetached being called with a non-retryable error.
-    const nonRetryableError = translate(new Error(`I break systems.`));
+    const nonRetryableError = new MessagingError(`I break systems.`);
```

Comment:
In the MessagingError class, `retryable` is set to true by default. So why this change?

Reply:
The next line sets retryable to false :)

Reply:
In the very next, we set the retryable flag to false.

Reply:
And previously, translate wasn't converting this to a MessagingError (it did in amqp-common)

Reply:
Yes to that.